src/NewProject/source_grid.cpp

```diff
@@ -351,19 +381,50 @@ bool sourceGrid::IsFileDataExit(filedata fdata) {
   return false;
 }
 
+QComboBox *sourceGrid::CreateLanguageCombo() {
+  auto combo = new QComboBox;
+  combo->addItem("BLIF", Design::Language::BLIF);
```

Comment:
Just wondering - should the user be able to change the language of, for example, *.vhdl to VERILOG 2001?
if we want to forbid it then we may want to fill the combo based on automatically deduced language within 'sibling' languages. For *.sv files only SV 2005 - SV 2017 would be available then.

Reply:
good point for backlog.